phpini/edit_vars.cgi

```diff
@@ -11,20 +11,21 @@ $conf = &get_config_as_user($in{'file'});
 print &ui_form_start("save_vars.cgi", "post");
 print &ui_hidden("file", $in{'file'}),"\n";
 print &ui_table_start($text{'vars_header'}, "width=100%", 4);
-
-print &ui_table_row($text{'vars_magic'},
-		    &onoff_radio("magic_quotes_gpc"));
-
-print &ui_table_row($text{'vars_runtime'},
-		    &onoff_radio("magic_quotes_runtime"));
-
-print &ui_table_row($text{'vars_register'},
-		    &onoff_radio("register_globals"));
-
-print &ui_table_row($text{'vars_long'},
-		    &onoff_radio("register_long_arrays"));
-
-print &ui_table_row($text{'vars_args'},
+if (php_version_test_maximum('7.4')) {
```

Comment:
Don't you need to pass the file into php_version_test_maximum ?

Reply:
No, because it will be caught automatically, from `$in{'file'}` if not set.

Reply:
It's a lot cleaner to pass `$in{'file'}` into this function as a parameter, rather than accessing %in from inside the function.

Reply:
> It's a lot cleaner to pass `$in{'file'}` into this function as a parameter, rather than accessing %in from inside the function.

Jamie, yes maybe, and we always expect external callers to pass file parameter. However, doing it inside of `phpini` module will clutter a lot of `phpini`-_pages_, as we will also have to pass it to `opt_help` sub _everywhere_. It will look ugly and cluttered ..

Reply:
Oh it's needed by `opt_help` as well? Hmm .. in that case, I suppose we have no choice, even though I don't really like using global variables.

Reply:
> I don't really like using global variables.

Yeah, I also try to avoid using globals anytime possible.